packages/flutter/test/material/dialog_test.dart

```diff
@@ -47,8 +47,8 @@ RenderParagraph _getTextRenderObjectFromDialog(WidgetTester tester, String text)
 }
 
 // What was the AlertDialog's ButtonBar when many of these tests were written,
-// is now a Container with an OverflowBar child. The Container's size and location
-// match the original ButtonBar's size and location.
+// is now a Padding widget with an OverflowBar child. The Padding widget's size
+// and location  match the original ButtonBar's size and location.
 Finder _findButtonBar() {
```

Comment:
Maybe more than the scope of this PR, but maybe we should rename this to something like `_findButtonPadding`, or `_findButtonContainer`?

Reply:
I tried making such a change. It's hard to understand why the tests are measuring the Padding widget that contains the OverflowBar (which contains the buttons) without reminding the reader that originally there was a Widget called "ButtonBar" that occupied the bounds of the space we're checking. Plus names like _findButtonPadding and _findButtonContainer are subtly misleading.